packages/docs/docs/components/select.mdx

```diff
@@ -36,6 +36,7 @@ Select component is powered by [react-select](https://react-select.com/home). AP
 | isMulti                   | Support multiple selected options                                  | boolean        | false   |
 | disabled                  |                                                                    | boolean        | false    |
 | size                      |                                                                    | `sm` `md` `lg` `xl`    | md      |
+| label                     |                                                                    | string         |         |
```

Comment:
label can also be react component right? I think `ReactNode` would be the right type, can we also make the change in `select.tsx`? @febinvarghese1